.github/CODEOWNERS

```diff
@@ -16,19 +16,19 @@
 *   @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
 # Schemas and schema file tooling
-/schemas/ @tigrannajaryan
-/internal/tools/schema_check.sh @tigrannajaryan
+/schemas/ @tigrannajaryan @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
+/internal/tools/schema_check.sh @tigrannajaryan @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
 # Logs semantic conventions
-/semantic_conventions/logs/ @tigrannajaryan
-/specification/logs/ @tigrannajaryan
+/semantic_conventions/logs/ @tigrannajaryan @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
+/specification/logs/ @tigrannajaryan @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
 
 # JVM semantic conventions approvers
-/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml @trask
-/semantic_conventions/metrics/process-runtime-jvm-metrics-experimental.yaml @trask
+/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml @open-telemetry/semconv-jvm-approvers @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers
```

Comment:
nit: might be easier to maintain if we put some indentation to align vertically - e.g. https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS